unity-environment/Assets/ML-Agents/Scripts/Monitor.cs

```diff
@@ -193,6 +189,7 @@ private float[] ToFloatArray(object input)
         }
         catch
         {
+            
```

Comment:
Emtpy catch blocks aren't ideal. What should we return in this case?

Reply:
If this is caught, it means that whatever we tried to convert to a float array could not be converted.

Reply:
Ok, the structure for this method ought to change, but lets not address in this PR.